packages/agent/src/agent/http/index.ts

```diff
@@ -269,7 +274,13 @@ export class HttpAgent implements Agent {
       this.#verifyQuerySignatures = options.verifyQuerySignatures;
     }
     // Default is 3
-    this._retryTimes = options.retryTimes ?? 3;
+    this.#retryTimes = options.retryTimes ?? 3;
+    // Delay strategy for retries. Default is exponential backoff
+    const defaultBackoffFactory = () =>
+      new ExponentialBackoff({
+        maxIterations: this.#retryTimes,
+      });
+    this.#backoffStrategy = options.backoffStrategy ?? defaultBackoffFactory;
```

Comment:
is the typescript compiler enough to ensure `options.backoffStrategy` isnt something other than a function?

Reply:
No, it wouldn't be prevented, but we aren't doing strict type enforcement on most options during construction in this code base at this stage

Reply:
in this case would `||` be a better fit in case someone provides e.g. `options.backoffStrategy = false` ?

Reply:
fair enough
```suggestion
    this.#backoffStrategy = options.backoffStrategy || defaultBackoffFactory;
```